test/Microsoft.Process.Environment.Tests/EnvironmentManagerWin32Tests.h

```diff
@@ -0,0 +1,15 @@
+﻿#pragma once
+
+namespace ProjectReunionCppTest
+{
+    class EnvironmentManagerWin32Tests {
+        BEGIN_TEST_CLASS(EnvironmentManagerWin32Tests)
+            TEST_CLASS_PROPERTY(L"ActivationContext", L"CppTest.dll.manifest")
```

Comment:
Ooooh. That's cute. I haven't see `ActivationContext` before. What's that do?

Reply:
Activation Context is for reg-free winrt.  This SxS manifest tells cppTest what dll to use and the class to use from the dll.

Once Dynamic Dependencies comes out I can switch this all out.

Reply:
One consequence (Maybe) of adding this dependency is that ProjectReunion_DLL will build anytime cpptest is being built.

Reply:
:-(

Don't add to CppTest. That really should be renamed AppLifecycleTest.

"Product" code cohabitates in ProjectReunion_DLL but for tests isolation is waaaay better. Adding multiple features into 1 test DLL creates unnecessary tying and friction. This is worse than product code as tests often do a lot more and, to be blunt, weird things. Making all that cohabitate nicely is harder than doing it for product code. It's also degrades the inner-loop - last thing I want to deal with is some having to rebuilding EVERY test when I'm iterating in 1 specific area.

Or to put it simply: Politicians take make small problems and roll them up into one big problem. Developers take one big problem and break it down into many small problems. Be a developer :P

Reply:
@DrusTheAxe how does this look?